### DISCUSSION TITLE
Add the `Expires` directive for `easylist-chat`

### DISCUSSION THREAD

**Discussion by partingscientist** [uBlockOrigin/uAssets#20732] [Ideas]
Currently, the list does not have it. I would assume this is not intentional, and the list would be updated every 4 days if my memory of the implementation is correct. Making it consistent with the other annoyances lists (7 days) is probably not a bad idea.

**Comment by Yuki2718** [comment]
> every 4 days

5 days if no directive

**Comment by ghost** [comment]
https://github.com/uBlockOrigin/uAssets/commit/a7634887aa157212659c1aec53af25b287511bd9 ?

**Reply by ghost** [reply]
Why it was removed? @gorhill ?

**Reply by ghost** [reply]
Hmmm, second header down below https://github.com/uBlockOrigin/uAssets/blame/ef88591a650732c5f719f42bea2ca2d30bcfc7ed/thirdparties/easylist/easylist-chat.txt#L19

**Reply by ghost** [reply]
Only the last line of the `template_header.txt` is included?

**Reply by ghost** [reply]
Three weeks ago header start to be trimmed https://github.com/uBlockOrigin/uAssets/commit/f05512e5d28645bf8846c3a285e4288110dae277#diff-18b417cd52de8250785338e6f28432d5ce6a5dd54e44342b3c65d06c664c887dR130-R137 

It seem the regex is not perfect.

***

Need a break, my eyes hurt.

**Reply by gorhill** [reply]
> Why it was removed?

I don't remember, might be just an oversight.
